docs/pages/management/admin/daemon.mdx

```diff
@@ -143,7 +153,33 @@ To upgrade a host to a newer version of Teleport, you must:
 
 </Admonition>
 
-## Further reading
+
+## Understanding Teleport daemon command line options for unit file configuration
+
+The `teleport install systemd` command includes a number of optional flags that you can use to strictly define parameters of the generated unit file and set where that output will be written.
+The following table includes all command line options available with the `teleport install systemd` command, a brief description of what they do, and our general recommended defaults tested on **Ubuntu 20.04 LTS**:
+
+
+| Flag | Description | General Recommended Default |
```

Comment:
Hmm, "General Recommended Default" sounds different than "Default". It makes it seem like the user should set these defaults if they don't know what else to put, when in reality Teleport will just do it for them.

Maybe change the column to "Default" instead?